transform/mattermost-analytics/models/marts/sales/hightouch/fct_cloud_trial_requests.sql

```diff
@@ -0,0 +1,29 @@
+with cloud_trial_requests as (
+    select 
+    email,
+    cws_installation
+    from 
+    {{ ref('int_cloud_trial_requests') }}
+    -- Fetch the most recent cloud trial
+    qualify row_number() over (partition by email order by trial_start_at desc) = 1
+)
+select
+    ctr.email,
+    l.lead_id as existing_lead_id,
+    '{{ var('cloud_enterprise_trial_campaign_id') }}' as campaign_id,
+    l.lead_id is not null as is_existing_lead,
+    cm.campaign_member_id is not null as is_existing_campaign_member,
+    -- Campaign member status
+    CASE 
+        WHEN ctr.cws_installation is not null then 'Workspace Created'
+        ELSE null
+    END as campaign_member_status,
+    -- Extra validation
+    {{ validate_email('ctr.email') }} as is_valid_email
+from
+    cloud_trial_requests ctr
+    left join {{ ref('stg_salesforce__lead') }} l on ctr.email = l.email
```

Comment:
Multiple leads with the same email might exist. For in-product trial request, only the oldest one is updated.